src/Native/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 2.8.12)
+
+project (IOT C)
+
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+
+set(CMAKE_SHARED_LIBRARY_PREFIX "")
+
+add_compile_options(-fPIC)
```

Comment:
What is this for?

Reply:
its saying generates Position Independent Code https://stackoverflow.com/questions/5311515/gcc-fpic-option

Reply:
this is usual for .so files, we should keep it